packages/calypso-color-schemes/package.json

```diff
@@ -25,7 +25,7 @@
 	},
 	"scripts": {
 		"clean": "npx rimraf js css src/__color-studio",
-		"prepare": "node bin/prepare-sass-assets.js && node bin/build-css.js"
+		"postinstall": "node bin/prepare-sass-assets.js && node bin/build-css.js"
```

Comment:
Not sure about this change tbh.

Calypso needs those commands before it can use the package, so `postinstall` seems natural. However any external consumer of `@automattic/calypso-color-schemes` will also run `postinstall`, which will fail if they have not installed the dev dependencies of the package.

We could keep `prepare` and bring back running `prepare` on Calypso's `postinstall`, but that means we have to use `lerna` just to run `prepare` in a couple of packages.

Reply:
Does `yarn workspaces run prepare` work to avoid `lerna`?

It looks like yarn uses different lifecycle scripts, should we be updating these? `prepare` is a holdover from npm days as far as I can tell…

Reply:
`yarn workspaces run prepare` fails if any package in the workspace does not implement a `prepare` script.

Yarn runs `prepare` (at least yarn v1), but only for the root package, it doesn't run it for the subpackages (https://github.com/yarnpkg/yarn/issues/3911).

So either we have our own explicit list (`"prepare": "yarn workspace @automattic/calypso-color-schemes run prepare && yarn workspace @automattic/languages run prepare"), use a different tool like [wsrun](https://www.npmjs.com/package/wsrun) or stick with `lerna`.